nss-cohort-36/bangazon-api-virtual_magicians
# Jw shooping cart

# Description
[Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.]
Closes #[Enter issue #]

## Type of change
Please delete options that are not relevant.

# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout jw-shoppingCart`
4. [Enter specific testing instructions here to approve]
check in postman to perform get request on orders and order_product
# Checklist:
